modules/ScreenNameLocalFormatter/build.gradle

```diff
@@ -0,0 +1,42 @@
+dependencies {
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "9.36.0"
```

Comment:
Some reviewers will ask you to keep the dependencies in alphabetical order.

Reply:
Fixed, `gw formatSource` has organized these lines alphabetically.